tests/suites/test_suite_psa_crypto.function

```diff
@@ -112,7 +112,7 @@ int lifetime_is_secure_element( psa_key_lifetime_t lifetime )
 {
     /* At the moment, anything that isn't a built-in lifetime is either
      * a secure element or unassigned. */
-    return( lifetime != PSA_KEY_LIFETIME_VOLATILE &&
+    return( ( ! PSA_KEY_LIFETIME_IS_VOLATILE( lifetime ) ) &&
```

Comment:
I can fix it but even in the full configuration (`MBEDTLS_PSA_CRYPTO_SE_C` enabled) `lifetime_is_secure_element` always returns `false`. I would be on the side of removing it but I don't know the whole story about it, let me know.

Reply:
It turns out that `test_suite_psa_crypto.data` doesn't have any test case with a key in a secure element. But if it did, `lifetime_is_[dynamic_]secure_element` would serve its purpose, which is (so far) to identify keys that have a slot number attribute.

Please leave this code around since it's incomplete but useful.

Reply:
ok fair enough, fixed and changed the name to `lifetime_is_dynamic_secure_element`.